src/Microsoft.Azure.ServiceBus/Amqp/AmqpMessageSender.cs

```diff
@@ -65,6 +86,75 @@ public override Task CloseAsync()
             }
         }
 
+        protected override async Task<long> OnScheduleMessageAsync(BrokeredMessage brokeredMessage)
+        {
+            // TODO: Ensure System.Transactions.Transaction.Current is null. Transactions are not supported by 1.0.0 version of dotnet core.
+            TimeoutHelper timeoutHelper = new TimeoutHelper(this.OperationTimeout, true);
+            using (AmqpMessage amqpMessage = AmqpMessageConverter.ClientGetMessage(brokeredMessage))
+            {
+                RequestResponseAmqpLink amqpLink =
+                    await
+                        this.RequestResponseLinkManager.GetOrCreateAsync(timeoutHelper.RemainingTime())
+                            .ConfigureAwait(false);
+
+                var request = AmqpRequestMessage.CreateRequest(
+                    ManagementConstants.Operations.ScheduleMessageOperation,
+                    this.OperationTimeout,
+                    null);
+
+                ArraySegment<byte>[] payload = amqpMessage.GetPayload();
+                BufferListStream buffer = new BufferListStream(payload);
+                ArraySegment<byte> value = buffer.ReadBytes((int)buffer.Length);
+
+                var entry = new AmqpMap();
+                {
```

Comment:
The API on the service side (broker) has a slightly different implementation for ScheduleMessageAsync(). These few lines are required just to take care of that. They will be updated when we update the service side code. 
Not fixing this for now.